packages/visualizations/app/styles/navi-visualizations/components/number-format-dropdown.less

```diff
@@ -4,6 +4,20 @@
  */
 
 .number-format-dropdown {
+  align-self: center;
+
+  &__trigger {
+    padding: 0 5px;
```

Comment:
Can we make the padding `5px` so it aligns with the `input` ?

Reply:
Do you mean `padding : 5px`?